jobs/JOBSPROC_RAP_DUMP

```diff
@@ -36,9 +36,14 @@ echo
 echo "####################################################################"
 echo
 echo "OBSPROC version is ${obsproc_ver:-not set}"
-echo "BUFR_UTIL version used by this job, for dump and dumplist, is \
-${bufr_util_ver:-not set}"
+echo "BUFR_DUMP version used by this job, for dump and dumplist, is \
+${bufr_dump_ver:-not set}"
 echo "grib_util module version used by this job is ${grib_util_ver:-not set}"
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)bufr_dump/") -eq 0 ]];
```

Comment:
@ShelleyMelchior-NOAA Nick's check is nicer than my:
echo HOMEbufr_dump is ${HOMEbufr_dump:?"###FATAL ERROR \"HOMEbufr_dump\" is not set"}
Shall I adopt it?

Reply:
@ilianagenkova Yes!